integrations/firehydrant/.port/resources/blueprints.json

```diff
@@ -0,0 +1,324 @@
+[
+    {
+        "identifier": "firehydrantIncident",
+        "description": "This blueprint represents a firehydrant incident",
+        "title": "FireHydrant Incident",
+        "icon": "Firehydrant",
```

Comment:
Please make all of the icons to be named as `FireHydrant`.
I will ask the team to add them to the system